common/v2/features/Dashboard/Settings/Import/Import.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { withRouter, Link, RouteComponentProps } from 'react-router-dom';
+// import { Button, Typography, Input } from '@mycrypto/ui';
+import styled from 'styled-components';
+
+import { ContentPanel } from 'v2/components';
+import { ImportBox, ImportSuccess } from './components';
+import { Layout } from 'v2/features';
+import { GlobalSettingsContext } from 'v2/providers';
+
+const Content = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+`;
+
+const CenteredContentPanel = styled(ContentPanel)`
+  width: 35rem;
+`;
+
+export interface PanelProps {
+  onBack(): void;
+  onNext(): void;
+}
+
+const steps = [ImportBox, ImportSuccess];
```

Comment:
Expecting to walk through two arrays with interdependent indexes is fragile and prone to errors.

Prefer:
```
const steps = [
 {
   heading: 'Import',
   component: ImportBox 
 },
 {
    heading: 'Import Complete',
    component: ImportSuccess 
 }
]; 
// .... 
  <CenteredContentPanel
          onBack={onBack}
          heading={steps[step].heading}
// ....
```

Add a step number if need be, or dynamically generate two arrays from your single source of truth.